lib/Echidna/Campaign.hs

```diff
@@ -346,9 +348,9 @@ campaign u v w ts d txs = do
   execStateT (evalRandT runCampaign g') (Campaign ((,Open (-1)) <$> if b then [] else ts) c mempty d' False (DS.fromList $ map (1,) txs) 0) where
     step        = runUpdate (updateTest v Nothing) >> lift u >> runCampaign
     runCampaign = use (hasLens . tests . to (fmap snd)) >>= update
-    update c    = view hasLens >>= \(CampaignConf tl sof _ q sl _ _ _ _ _) ->
+    update c    = view hasLens >>= \(CampaignConf tl sof _ q sl _ _ _ _ _) -> get >>= \(view hasLens -> Campaign _ _ _ _ _ _ sq) ->
```

Comment:
Add a new line after `>>=` and use `NamedFieldPuns` to extract `sq` from Campaign.